{
  "path": "/documentation/foundation/recoverableerror/2963386-attemptrecovery",
  "type": "Instance Method",
  "name": "attemptRecovery(optionIndex:resultHandler:)",
  "desc": "No overview available.",
  "items": [],
  "declaration": "func attemptRecovery(optionIndex recoveryOptionIndex: Int, resultHandler handler: @escaping (_ recovered: Bool) -> Void)"
}